tools/plugins/com.liferay.ide.project.core/src/com/liferay/ide/project/core/modules/NewLiferayModuleProjectOpMethods.java

```diff
@@ -93,8 +96,13 @@ public static final Status execute( final NewLiferayModuleProjectOp op, final Pr
                 }
             }
 
+            if( op.getAddJspValidationSupport().content() )
+            {
+                ModuleCoreUtil.addFacetsIfNeeded( projectLocation.toFile(), new NullProgressMonitor() );
```

Comment:
Does this always add the web/jsp validation?

Reply:
Yes, this way is we already done before:
right click on project -> configure -> Add Jsp valistion support